demos/bio/build.info

```diff
@@ -0,0 +1,29 @@
+PROGRAMS{noinst}=client-arg sconnect server-cmod client-conf saccept server-arg server-conf
```

Comment:
You did remember it in a few places, like here here, but......

Reply:
I've reconciled all of these to be {noinst}